Assets/MixedRealityToolkit/InputSystem/Sources/DictationInputManager.cs

```diff
@@ -67,6 +68,7 @@ public class DictationInputManager : BaseInputSource
 
         public override InputType[] Capabilities => new[] { InputType.Voice };
 
+
```

Comment:
We can probably revert any changes in this class.

Reply:
ok